Setup/UpgradeData.php

```diff
@@ -271,4 +276,42 @@ private function createProductAttributes()
             );
         }
     }
+
+    private function addPlanIdFieldProductAttribute()
+    {
+        /** @var EavSetup $eavSetup */
+        $eavSetup = $this->eavSetupFactory->create(['setup' => $this->moduleDataSetup]);
+
+        $attribute = $eavSetup->getAttribute(Product::ENTITY, 'vindi_plan_id');
+        if (!$attribute) {
+            $eavSetup->addAttribute(Product::ENTITY, 'vindi_plan_id', [
+                'sort_order' => 1,
+                'type' => 'varchar',
+                'backend' => '',
+                'frontend' => '',
+                'label' => 'Use Registered Plan in Vindi',
```

Comment:
Seguindo o outro comentário, aqui ficaria `Vindi Plan`. o que acha? :thinking: 
```suggestion
                'label' => 'Vindi Plan',
```